frontend/src/app/sessions/user-sessions-wrapper/create-session/provision-workspace/provision-workspace.component.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { Router } from '@angular/router';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { filter } from 'rxjs';
+import {
+  Model,
+  ModelService,
+} from 'src/app/projects/models/service/model.service';
+import { ProjectService } from 'src/app/projects/service/project.service';
+import { SessionService } from 'src/app/sessions/service/session.service';
+import { UserSessionService } from 'src/app/sessions/service/user-session.service';
+
+@UntilDestroy()
+@Component({
+  selector: 'app-provision-workspace',
+  templateUrl: './provision-workspace.component.html',
+  styleUrls: ['./provision-workspace.component.css'],
+})
+export class ProvisionWorkspaceComponent implements OnInit {
+  projectSlug?: string;
+  projectType?: string;
```

Comment:
Here I would either again use the `*ngIf="(projectService.project$ | async)?.type === 'training'"` in the template, or just store the whole project in here and use `project.type` and `project.slug`

Reply:
I'll store the project then, since we need that when requesting a session anyway.